internal/pkg/otel/samples/darwin/logs_metrics_traces.yml

```diff
@@ -123,8 +104,8 @@ service:
 
     metrics/hostmetrics:
       receivers: [hostmetrics/system]
-      processors: [elasticinframetrics, resourcedetection, attributes/dataset, resource/process]
-      exporters: [elasticsearch/ecs]
```

Comment:
This change is described as a "deprecation", but isn't this a breaking change? This looks like data that was previously sent in ECS format will now be sent in OTel format, which changes how the data can be accessed in Elasticsearch - is my understanding correct?